src/docs/intro.html

```diff
@@ -197,6 +199,19 @@ <h3 id="community">Community</h3>
               Feel free to reach out to any of the current moderators if you'd like to help out.
             </p>
 
+            <hr class="hr-dash my-7">
+            
+            <h3 id="nontechnical">A non technical explanation of Plasma</h3>
```

Comment:
Might help to explain why Plasma or what it does to a sidechain (i.e. economic protection)